src/main/java/core/basesyntax/generator/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.generator;
+
+import core.basesyntax.db.Storage;
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    @Override
+    public String getReport() {
+        if (Storage.fruits.isEmpty()) {
+            return "No records to report";
+        }
+        StringBuilder report = new StringBuilder();
```

Comment:
Consider adding a header to the report (e.g., 'Fruit,Quantity') for better clarity and consistency.